src/foam/java/refinements.js

```diff
@@ -554,7 +554,78 @@ foam.CLASS({
   properties: [
     ['javaType', 'java.lang.Enum'],
     ['javaInfoType', 'foam.core.AbstractFObjectPropertyInfo'],
-    ['javaJSONParser', 'foam.lib.json.FObjectParser']
+    ['javaJSONParser', 'foam.lib.json.IntParser']
+  ],
+  methods: [
+    function createJavaPropertyInfo_(cls) {
+      var info = this.SUPER(cls);
+
+      const OF = (this.of ? this.of.id ? this.of.id : this.of : null);
+
+      info.method({
+        name: 'of',
+        visibility: 'public',
+        type: 'String',
```

Comment:
Until primitive types have support for nested arrays, it could be consistently returning a string for now.